### PR TITLE
Update imports and fix spacing in stacktrace.clj

### PR DESCRIPTION
**Pull request by sirmspencer**

### CODE REVIEW

**Comment by weavejester** [comment]
Can you split this up into two commits? Each commit should make one change, so for instance here we'd have a commit that fixes the indentation, and a commit that fixes the require form.

Also, I think as long as we're fixing this, we should change the stacktrace require forms to use `:as` rather than `:refer`, e.g.

    [stacktrace.core :as st]
    [stacktrace.repl :as repl]

**Comment by sirmspencer** [comment]
Updating imports looks like a bigger task than I was thinking.  Ill leave it for its own task.

**Comment by sirmspencer** [comment]
@weavejester Can you look at this again?

File: ring-devel/src/ring/middleware/stacktrace.clj [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 (defn wrap-stacktrace-log
   "Wrap a handler such that exceptions are logged to *err* and then rethrown.
+
```
Comment (weavejester) [inline ring-devel/src/ring/middleware/stacktrace.clj:16]: What's the purpose of this newline?

Comment (sirmspencer) [reply ring-devel/src/ring/middleware/stacktrace.clj:16]: It matches the formatting on line 112 of the same file, and line 25 of reload.clj in the same directory.

**Pull request merged**